Update the About the release page - For M1 (#6432)
wso2/docs-apim: en/docs/get-started/about-this-release.md

## New features

??? note "JDK 17 support"

>     You can now run our products with JDK 17

Reviewer: ```suggestion
    You can now run our products with JDK 17.
```

??? note "OpenTelemetry Support"

>     OpenTelemetry combines OpenTracing and OpenCencus which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics and logs are collected, transmitted and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backend

Reviewer 2: ```suggestion
    OpenTelemetry combines OpenTracing and OpenCencus which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics and logs are collected, transmitted and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backends
```

>     OpenTelemetry combines OpenTracing and OpenCencus which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics and logs are collected, transmitted and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backends

Reviewer: ```suggestion
    OpenTelemetry combines OpenTracing and OpenCencus, which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics, and logs are collected, transmitted, and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backends.
```

??? note "ELK Support for MI"

>     From WSO2 API Manager 4.2.0 onwards, Micro Integrator now enables publishing operational analytics for Elastic stack as a replacement for WSO2 EI Analytics.

Reviewer: ```suggestion
    From WSO2 API Manager 4.2.0 onwards, Micro Integrator now enables publishing operational analytics for Elastic Stack as a replacement for WSO2 EI Analytics.
```

??? note "Configuring Self Sign up through Admin Portal"

>     WSO2 API Manager now offers you the ability to self-signup into the Developer Portal as well as the option to disable self-signup. You can also migrate the self-signup configurations from the registry through the **Advance Configuration** option or through editing the `tenant-conf.json` file.

Reviewer 2: This should change a little. The self signup capability is already there in the Devportal. What comes in as a new feature is the ability to configure self signup through the admin portal as opposed to the configuration we did from the Management console.

??? note "Configuring API Lifecycle through Admin Portal"

>     APIs created in WSO2 API Manager have their own life cycle consisting of the following: a set of life cycle states, specific actions for each state transition, and a checklist of items before a state transition occurs. An API has a predefined life cycle consisting of six states by default. This feature allows you to customize and/or add new states, migrate API Lifecycle through the **Advance Configuration** option or through editing the `tenant-conf.json` file.

Reviewer 2: Let's not mention migrate. We can say configure.

??? note "Enable logs without server restart"

>     In this new version of WSO2 API Manager, you can enable or disable correlation logs as needed for troubleshooting at runtime, as a server restart is not required.

Reviewer 2: This feature is currently available for Micro Integrator only

??? note "Removal of Jaggery Apps from Portals"

>     Jaggery is not currently in an active stage of development or upkeep. Therefore, it is essential to switch to a different language which is closer to server-side while still using the react component as the client-side handler.

Reviewer 2: now JSP is used as the server side language for all portals. All jaggeryapps have been migrated to webapps.

??? note "Registry artifact view support for MI Management API/Dashboard"

>     A new page has been added to the MI-Dashboard to view the `<MI-HOME>/registry` directory and registry metadata, properties and content.

Reviewer: ```suggestion
    A new page has been added to the Micro Integrator dashboard to view the `<MI-HOME>/registry` directory and registry metadata, properties, and content.
```

    REST API resources have been added to the Micro Integrator as a **management API** to facilitate the functionalities of fetching existing registry data, adding new registry artifacts, modifying and deleting existing registries.

??? note "Gateway Support for Single Usage Token"

>     The JWT access tokens supplied by the gateway can be used to access the dedicated API resource as many times as the user requires. However, if the application developer wishes to limit this functionality by allowing just one chance to access the resource, One Time Token can be utilized. A One Time Token (JWT) is a JWT that will be revoked after a single use. This functionality is only accessible for JWT access tokens.

Reviewer: ```suggestion
    The JWT access tokens supplied by the Gateway can be used to access the dedicated API resource as many times as the user requires. However, if the application developer wishes to limit this functionality by allowing just one chance to access the resource, One Time Token can be utilized. A One Time Token (JWT) is a JWT that will be revoked after a single use. This functionality is only accessible for JWT access tokens.
```

> ??? note "Import/Export of Operation Policies with APICTL"

Reviewer 2: Let's rename the title to "Support for managing Common API Policies with APICTL"

Reviewer: ```suggestion
??? note "Support for managing Common API Policies with APICTL"
```

    As an upgrade from the current version of APICTL (4.1.0), a command to export the shared operation policies in particular has been added.

??? note "Handling accidental deletions of shared backend certificates"

>     Backend certificates are currently shared amongst APIs based on endpoint. As a result, another person who creates an API with the same endpoint can view and remove a certificate that does not belong to the user. As a result, this new functionality is designed to limit the ability to delete backend certificates.

Reviewer: ```suggestion
    Backend certificates are currently shared amongst APIs based on the endpoint. As a result, another person who creates an API with the same endpoint can view and remove a certificate that does not belong to the user. This new functionality is designed to limit the ability to delete backend certificates.
```

??? note "API Linters for validating the API Specifications"
    This feature integrates an API Linter with the support of custom rules.

??? note "Generate API Documentation Automatically from OAS"
    This feature generates API Documentation automatically using the API’s Open API Specification.

> ??? note "Throttle policy export import support for API-M"

Reviewer 2: Let's rename the title to "Support for managing Rate Limiting Policies with APICTL"

>     This new capability allows you to import and export throttle policies between environments. Throttling policies are imported to the target environment in a separate stage before importing either the API or the App. After importing throttling policies to the target environment, you may import the API/Application without encountering any throttling policy support problems.

Reviewer: ```suggestion
    This new capability allows you to import and export rate limiting policies between environments. Rate limiting policies are imported to the target environment in a separate stage before importing either the API or the App. After importing rate limiting policies to the target environment, you may import the API/Application without encountering any rate limiting policy support problems.
```

> ??? note "Introducing MI control plane, Search and Pagination"

Reviewer: This is incorrect I think. This change involves the removal of a database to handle backend stuff in the dashboard plus an improved search and pagination for MI dashboard and the Management API. Nothing there about a new Control Plane as far as I know.

Author: @tharikaGitHub Please confirm the comment about this feature. 

Thanks

Reviewer 2: We can change the title as "MI Dashboard Revamp and Pagination and Search capability for MI Management API"

    This feature introduces a new Micro Integrator Control plane and Implement Pagination and Search Functionality.

## Compatible WSO2 product versions

- WSO2 API Manager 4.2.0 is compatible with WSO2 Identity Server 6.0.0.

> - WSO2 API Manager 4.2.0 is compatible with Choreo Connect {{choreo_connect.version}}.

Reviewer: I don't think this is correct because APK is coming in and CC is going out of the docs. Let's check with Pubudu and remove.

Author: Keeping this, because we have not removed CC docs yet. Let this be as a placeholder. WDYT?

Reviewer: Ok, +1. We can change this once removed.

> ## Fixed issues

Reviewer: @tharikaGitHub we need to add the link for fixed issues, known issues, etc. Can you provide this?

Reviewer 2: Link for fixed issues - https://github.com/wso2/api-manager/issues?q=is%3Aissue+milestone%3AAPIM-4.2.0-M1+is%3Aclosed

Link for known issues - https://github.com/wso2/api-manager/issues?q=is%3Aissue+is%3Aopen
